utils/data_manager.py

```diff
@@ -1,7 +1,15 @@
 import os
 
 def getImagenetData(dataset):
-    if dataset == 'imagenet' or dataset == 'imagenet-256px-q90':
+
+    if dataset == 'imagenet':
+        os.system('mkdir -p data')
+        if not os.path.exists('./data/imagenet1k-train.rec'):
+            os.system("aws s3 cp  s3://imagenet-rec/imagenet1k-train.rec data/")
+        if not os.path.exists('./data/imagenet1k-val.rec'):
+            os.system("aws s3 cp  s3://imagenet-rec/imagenet1k-val.rec data/")
+
+    elif dataset == 'imagenet-256px-q90':
         if not os.path.exists(os.path.expanduser('~/data/imagenet1k-train.rec')):
```

Comment:
This clause is not needed. I think, dataset=='imagenet-256px-q90' means the "imagenet" dataset we were using before.  We can add it to the above if clause